server/prisma/migrations/20230324125932_create_instance_settings/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- CreateTable
+-- We aren't generating ids automatically, because the instance will use one record.
+-- But we allow for multiple record, because we can add the ability for instance owner,
+-- to save multiple settings, and use them as templates.
+CREATE TABLE "instance_settings" (
+    "id" SERIAL NOT NULL,
```

Comment:
Ah, that's a little different. If the plan is to have multiple records, then we don't need to do anything special. It's fine to use `@default(autoincrement())` and there's nothing weird that demands explanation.
```suggestion
CREATE TABLE "instance_settings" (
    "id" SERIAL NOT NULL,
```